manifests/o/Obsidian/Obsidian/1.10.6/Obsidian.Obsidian.installer.yaml

```diff
@@ -14,8 +13,6 @@ FileExtensions:
 - md
 ProductCode: bd400747-f0c1-5638-a859-982036102edf
 ReleaseDate: 2025-11-25
-AppsAndFeaturesEntries:

```

Comment:
If you revert the changes to the installer file, it will have an easier time passing validation